src/components/treePopup/index.tsx

```diff
@@ -149,31 +154,48 @@ const TreePopup: React.FC<TreePopupProps> = ({
               )}
               <GreyText>{treeInfo.address}</GreyText>
               {(() => {
-                switch (treeInfo.id) {
-                  case NO_TREE_PRESENT:
+                let editSiteButton = <div/>;
+                if (userIsAdmin) {
+                editSiteButton = <GreenLinkButton
+                                  to={`${ParameterizedRouteBases.SITE}${treeInfo.id}`}
+                                  state={{ destination: Routes.MY_TREES }}
+                                  target="_blank"
+                                >
+                                  Edit Site Page
+                                </GreenLinkButton>
+                                }
+
+
+                if (!treeInfo.treePresent) {
+                  
                     return (
-                      <PlantRequest>
-                        Want to plant a tree here?{' '}
-                        <Typography.Link
-                          href={CITY_PLANTING_REQUEST_LINK}
-                          target="_blank"
-                        >
-                          Submit a request to the city!
-                        </Typography.Link>
-                      </PlantRequest>
+                      <div>
+                        <PlantRequest>
+                          Want to plant a tree here?{' '}
+                          <Typography.Link
+                            href={CITY_PLANTING_REQUEST_LINK}
+                            target="_blank"
+                          >
+                            Submit a request to the city!
+                          </Typography.Link>
+                        </PlantRequest>
+                        {editSiteButton}
+                      </div>
                     );
-                  default:
+                }
                     return (
-                      <GreenLinkButton
-                        to={`${ParameterizedRouteBases.TREE}${treeInfo.id}`}
-                        state={returnState}
-                        target="_blank"
-                      >
-                        More Info
-                      </GreenLinkButton>
+                      <div>
+                        <GreenLinkButton
+                          to={`${ParameterizedRouteBases.TREE}${treeInfo.id}`}
+                          state={returnState}
+                          target="_blank"
+                        >
+                          More Info
+                        </GreenLinkButton>
+                        <span style={{ marginLeft: '.1rem' }}> {editSiteButton} </span>
```

Comment:
Prefer to do conditional rendering as something like
```jsx
{userIsAdmin && 
<GreenLinkButton
     to={`${ParameterizedRouteBases.SITE}${treeInfo.id}`}
     state={{ destination: Routes.MY_TREES }}
     target="_blank"
    >
Edit Site Page
</GreenLinkButton>
}
```
instead of inserting empty div if not admin